src/main/resources/TourCode/AjaxActions.xml

```diff
@@ -0,0 +1,141 @@
+     #if ($bootstrapTourDoc)
+        ## Save step
+        #set($title = $request.getParameter("title"))
+        #set($path = $request.getParameter("path"))
+        #set($element = $request.getParameter("element"))
+        #set($backdrop = $request.getParameter("backdrop"))
+        #set($content = $request.getParameter("content"))
+        #set($placement = $request.getParameter("placement"))
+        #set($nbSteps = $request.getParameter("nbSteps"))
+        #set($stepOrder = $request.getParameter("stepOrder"))
+        #set($isNewStep = true)
+        #if ("$!stepOrder" != '') ## Edit
+            #set($stepObj = $bootstrapTourDoc.getObject('TourCode.StepClass','order',$stepOrder))
+            #if($stepObj)
+               #set($isNewStep = false)                
+            #end
+        #end
+        #if ($isNewStep)
+           ## New step
+           #set($stepObj = $bootstrapTourDoc.newObject('TourCode.StepClass'))
+        #end
+        #set($discard = $stepObj.set('path', $path))
+        #set($discard = $stepObj.set('element', $element))
+        #set($discard = $stepObj.set('title', $title))
+        #set($discard = $stepObj.set('placement', $placement))
+        #set($discard = $stepObj.set('backdrop', $backdrop))
+        #set($discard = $stepObj.set('content', $content))
+        #if($isNewStep)
+           #set($discard = $stepObj.set('order', $nbSteps))
+        #end
+        ## Add the javascriptExtention object that contains the Bootstrap tour js code call
+        #set($jseObj = $bootstrapTourDoc.getObject('XWiki.JavaScriptExtension'))
+        #if (!$jseObj)
+           ## Create the object
+           #set($jseObj = $bootstrapTourDoc.newObject('XWiki.JavaScriptExtension'))
+           #set($discard = $jseObj.set('name', 'BootstraptTour js call'))
+           #set($discard = $jseObj.set('code', $xwiki.getDocument("TourCode.BSTCall").getContent()))
+           #set($discard = $jseObj.set('use', 'onDemand'))
+           #set($discard = $jseObj.set('parse', 1))
+           #set($discard = $jseObj.set('cache', 'long'))
+        #end
+        ## Add the styleSheet object that contains the Bootstrap tour CSS code call
+        #set($sseObj = $bootstrapTourDoc.getObject('XWiki.StyleSheetExtension'))
+        #if (!$sseObj)
+           ## Create the object
+           #set($sseObj = $bootstrapTourDoc.newObject('XWiki.StyleSheetExtension'))
+           #set($discard = "$sseObj.set('name', 'BootstraptTour css call')")
+           #set($cssURL = '$xwiki.getURL("TourCode.BootstrapTourPlugin", "ssx")')
+           #set($discard = $sseObj.set('code', "@import '$cssURL'"))
+           #set($discard = $sseObj.set('use', 'currentPage'))
+           #set($discard = $sseObj.set('parse', 1))
+           #set($discard = $sseObj.set('cache', 'long'))
+        #end
+        #set($discard = $bootstrapTourDoc.save())
+        #set($BSTDocRef = $bootstrapTourDoc)
+        #includeForm("TourCode.StepsListing")
```

Comment:
AFAIR, `#includeForm` is deprecated. You should use the `{{include}}` macro instead.